scripts/make_tor_targets.py

```diff
@@ -0,0 +1,114 @@
+import json
```

Comment:
Also, it would be probably worth mentioning in some readme/manual for deployment what one needs to do when their goal is to update the list of tor bridges we use. (Or, is there some plan to perform this kind of update automatically when we're deploying? Would that be safe and desirable?)

Reply:
In an ideal world this would be done automatically and periodically on the deployed host. Since I wasn't sure if we wanted to do this I implemented it for the time being as something that somebody has to manually run.

If we are to require manually running it, we should document it's usage in the backend docs, though I am not sure what is the correct place to do so.

We currently have backend related documentation in the following places:
- https://github.com/ooni/backend/wiki
- https://ooni.github.io/pipeline/
- https://github.com/ooni/pipeline#ooni-backend

It's unclear to me which one is the latest and where I should be putting this documentation.

cc @FedericoCeratto